### PR TITLE
🔥 [HOTFIX]🔥 Fixed display issue on multiple relationships

### LINKED ISSUE #4528: PR #4417 broke relationship display of more than one value in metadata
Pull Request #4417 broke the rendering of Relationship links for properties that have more than one value.

Previously, relationships went through the `default` in `Metadata.js`.  This process took into account that values should be rendered with `<ValueList>` with interposed a comma between values when there is more than one.

This is no longer used and values with more than one relationship are rendered next to each other with no separation.  This should have been an easy catch if the dev and QA tests would have been done with more than one value.

cc @grafitto

### PR DESCRIPTION
**Pull request by grafitto**
fixes #4528 

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

File: app/react/Metadata/components/RelationshipLink.tsx [hunk 1]
```diff
@@ -27,26 +27,25 @@ const connector = connect(mapStateToProps, mapDispatchToProps);
...
-      );
-    }
+const RelationshipLink = ({ propValue: propVal, selectConnection }: mappedProps) => {
+  if (propVal.relatedEntity) {
     return (
-      <I18NLink key={prop.url} to={prop.url}>
-        {prop.icon && <Icon className="item-icon" data={prop.icon} />}
-        {prop.value}
-      </I18NLink>
+      <a
```
Comment (RafaPolit) [inline app/react/Metadata/components/RelationshipLink.tsx]: This is not correct.  `a` tags should be links.  If you are only looking for an onClick event, this should be a button.

File: app/react/Metadata/components/specs/__snapshots__/Metadata.spec.js.snap [hunk 2]
```diff
@@ -225,15 +240,34 @@ exports[`Metadata should render links when the property has url 1`] = `
...
+                "5": "g",
+                "6": " ",
+                "7": "v",
+                "8": "a",
+                "9": "l",
+                "value": <Connect(RelationshipLink)
+                  propValue="string value"
+                />,
+              },
+            ],
```
Comment (RafaPolit) [inline app/react/Metadata/components/specs/__snapshots__/Metadata.spec.js.snap]: This cannot be correct.  Either you are passing the wrong value to the test, or this means that you are passing each letter of a string like the array values.  Please either fix the text or the implementation.

File: app/react/Metadata/components/specs/__snapshots__/Metadata.spec.js.snap [hunk 3]
```diff
@@ -266,7 +301,25 @@ exports[`Metadata should render links with icons if propery has url and icon 1`]
...
+                "3": "i",
+                "4": "n",
+                "5": "g",
+                "6": " ",
+                "7": "v",
+                "8": "a",
+                "9": "l",
+                "value": <Connect(RelationshipLink)
+                  propValue="string value"
+                />,
```
Comment (RafaPolit) [inline app/react/Metadata/components/specs/__snapshots__/Metadata.spec.js.snap]: Same as above... why is the string being made into an array?

File: app/react/Metadata/components/Metadata.js [hunk 4]
```diff
@@ -7,6 +8,16 @@ import GeolocationViewer from './GeolocationViewer';
 import { RelationshipLink } from './RelationshipLink';
 import ValueList from './ValueList';
 
+const renderRelationshipLinks = linksProp => {
+  const formattedLinkValues = Array.isArray(linksProp.value) ? linksProp.value : [linksProp.value];
```
Comment (RafaPolit) [inline app/react/Metadata/components/Metadata.js:12]: Can the value in relationships really be something that is not an array? I thought single values were coming as arrays nonetheless.  Can you confirm this please?

File: app/react/Metadata/components/RelationshipLink.tsx (hunk 1 above)
Comment (grafitto) [reply app/react/Metadata/components/RelationshipLink.tsx]: Replaced

File: app/react/Metadata/components/specs/__snapshots__/Metadata.spec.js.snap (hunk 2 above)
Comment (grafitto) [reply app/react/Metadata/components/specs/__snapshots__/Metadata.spec.js.snap]: Fixed

File: app/react/Metadata/components/specs/__snapshots__/Metadata.spec.js.snap (hunk 3 above)
Comment (grafitto) [reply app/react/Metadata/components/specs/__snapshots__/Metadata.spec.js.snap]: Fixed

File: app/react/Metadata/components/Metadata.js (hunk 4 above)
Comment (grafitto) [reply app/react/Metadata/components/Metadata.js:12]: In some cases, the value is a string, based on the existing tests

File: app/react/App/scss/modules/_metadata-view.scss [hunk 5]
```diff
@@ -50,6 +50,19 @@
           content: ',';
         }
       }
+
+      button.link-button {
+        background: none !important;
+        border: none;
+        padding: 0 !important;
+        color: #2b56c1;
```
Comment (RafaPolit) [inline app/react/App/scss/modules/_metadata-view.scss]: I believe we have some sort of variables for link colors?

File: app/react/App/scss/modules/_metadata-view.scss [hunk 6]
```diff
@@ -50,6 +50,19 @@
...
+        cursor: pointer;
+        background-color: transparent;
+        
+        &:hover {
+          text-decoration: underline;
+        }
+      }
```
Comment (RafaPolit) [inline app/react/App/scss/modules/_metadata-view.scss]: Maybe we already have a "button zeroing out" class somewhere?

Comment (grafitto) [reply app/react/App/scss/modules/_metadata-view.scss]: I couldn't find any

File: app/react/App/scss/modules/_metadata-view.scss (hunk 5 above)
Comment (grafitto) [reply app/react/App/scss/modules/_metadata-view.scss]: Found

**Pull request merged**